modules/ROOT/pages/veevavault/veevavault-connector-design-center.adoc

```diff
@@ -0,0 +1,60 @@
+= VeevaVault Connector Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Design Center's Flow Designer enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Flow Designer field values, see the xref:./veevavault-connector-reference.adoc[VeevaVault Connector Reference].
+
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click the trigger card.
+. If you are using the HTTP Listener operation as a trigger, search for and select HTTP Listener.
+. In the HTTP Listener’s General tab, enter the path-absolute-URL that defines the resource location.	
+. Click off the trigger card.
+
+Flow Designer automatically saves all changes.
+
+== Configure the Target Component
+
+The target component enables your app to process data received from the trigger. VeevaVault Connector provides operations that help you integrate VeevaVault with third party applications and services. For example, this integration enables you to creates vault connectivity and retrieve picklists data from Vault, which can be further integrated or process.
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the Veeva Vault Connector.
+. Select the connector name.
+. Select an operation for the connector, such as *Get Picklists*.
+. If needed, specify values that are required in the General tab. For the Get Picklists operation, enter the required Picklist Name value whose details you want to retrieve.
+. Select Click here to set it up to set up other required fields.
+. Enter Configuration Name.
+. Click Add Connection and enter connection name.
+. In Connection Type, Select *1-Basic Authentication*.
+. In General tab, enter below required fields:
++
+[cols=",",options="header",]
+|===
+|*Field* |*Description* | Required?
+|*Name* |Enter a name for this configuration to reference it later. |Yes
+|*Vault URL* |Enter Veeva Vault instance URL. |Yes
+|*Username* |Enter the Veeva Vault instance username. |Yes
+|*Password* |Enter the corresponding password. |Yes
+|*Version* |Enter the Veeva Vault instance version (e.g: v19.1). |Yes
+|*Client Id* |Enter Client Id in format *\{organisation}-{company\|team}* to send with each request in header for tracking purpose. The Client Id is prefix with *"mulesoft"* and suffix with *"client-vaultconnector"* internally. E.g : *mulesoft-abcpharma-clinicalprod-client-vaultconnector*. |Yes
```

Comment:
```suggestion
|*Client Id* |Enter a client ID, in the format *\{organisation}-{company\|team}*, to send in each request header, for tracking. 
Prefix this client ID with *"mulesoft"* and end it with with *"client-vaultconnector"* internally: *mulesoft-abcpharma-clinicalprod-client-vaultconnector*. |Yes
```

Reply:
Do we need to explain "internally"? Are there two different IDs: one internal and one external?